acos_client/tests/t.py

```diff
@@ -187,6 +195,8 @@ def run_all(ax, partition, pmap):
         c.system.information()
     except acos_client.errors.AuthenticationFailure:
         print("got bad auth exception, good")
+    except acos_client.errors.ACOSException:
+        print("got bad auth exception, good")
```

Comment:
I took a peak at this. The error handling upstream isn't catching the auth error as they changed the unauthorized logic to return more consistent results.

We need to add the error code (it gets spit out with the exception) to responses.py and handle that properly. I'll do that in another patch. This won't be needed after.

Reply:
I cleaned this up and rebased with PR #228